code-of-conduct-languages/hi.md

```diff
@@ -1,99 +1,66 @@
-जो प्रोजेक्ट अनुरक्षक आचार संहिता का पालन नहीं करते या लागू नहीं करते, उन्हें प्रोजेक्ट टीम से अस्थायी या स्थायी रूप से हटाया जा सकता है।
+परियोजना अनुरक्षकों के पास इस आचार संहिता के अनुरूप नहीं होने वाली टिप्पणियों, प्रतिबद्धताओं, कोड, विकी संपादनों, मुद्दों और अन्य योगदानों को हटाने, संपादित करने या अस्वीकार करने का अधिकार और जिम्मेदारी है। इस आचार संहिता को अपनाकर, परियोजना अनुरक्षक CNCF परियोजना के प्रबंधन के हर पहलू पर इन सिद्धांतों को निष्पक्ष रूप से और लगातार लागू करने के लिए प्रतिबद्ध हैं। जो प्रोजेक्ट अनुरक्षक आचार संहिता का पालन नहीं करते या लागू नहीं करते, उन्हें प्रोजेक्ट टीम से अस्थायी या स्थायी रूप से हटाया जा सकता है।
 
 ## रिपोर्टिंग
 
-कुबेरनेट्स समुदाय में होने वाली घटनाओं के लिए, <conduct@kubernetes.io> के माध्यम से [कुबेरनेट्स आचार संहिता समिति](https://git.k8s.io/community/committee-code-of-conduct) से संपर्क करें। आप तीन व्यावसायिक दिनों के भीतर प्रतिक्रिया की उम्मीद कर सकते हैं।
+कुबेरनेट्स समुदाय में होने वाली घटनाओं के लिए, [conduct@kubernetes.io](mailto:conduct@kubernetes.io)<span style="text-decoration:underline;"> [कुबेरनेट्स आचार संहिता समिति](https://git.k8s.io/community/committee-code-of-conduct)</span> से संपर्क करें। आप तीन व्यावसायिक दिनों के भीतर जवाब प्राप्त करने की उम्मीद कर सकते हैं।
 
-अन्य परियोजनाओं के लिए, या ऐसी घटनाओं के लिए जो परियोजना-अज्ञेयवादी हैं या एकाधिक सीएनसीएफ पी को प्रभावित करती हैं, कृपया संपर्क करें  [CNCF आचार संहिता समिति](https://www.cncf.io/conduct/committee/) और conduct@cncf.io के जरिए। वैकल्पिक रूप से, आप अपनी रिपोर्ट सबमिट करने के लिए [CNCF आचार संहिता समिति](https://www.cncf.io/conduct/committee/) के किसी भी व्यक्तिगत सदस्य से संपर्क कर सकते हैं। रिपोर्ट सबमिट करने के तरीके के बारे में अधिक विस्तृत निर्देशों के लिए, जिसमें गुमनाम रूप से रिपोर्ट सबमिट करने का तरीका भी शामिल है, कृपया हमारी [घटना समाधान प्रक्रियाएँ](https://www.cncf.io/conduct/procedures/) देखें। आप तीन व्यावसायिक दिनों के भीतर प्रतिक्रिया की उम्मीद कर सकते हैं।
+अन्य परियोजनाओं के लिए, या ऐसी घटनाओं के लिए जो परियोजना-अज्ञेयवादी हैं या कई CNCF परियोजनाओं को प्रभावित करती हैं, कृपया [conduct@cncf.io](mailto:conduct@cncf.io) पर [CNCF आचार संहिता समिति](https://www.cncf.io/conduct/committee/) से संपर्क करें<span style="text-decoration:underline;">।</span> वैकल्पिक रूप से, आप अपनी रिपोर्ट सबमिट करने के लिए [CNCF आचार संहिता समिति](https://www.cncf.io/conduct/committee/) के किसी भी व्यक्तिगत सदस्य से संपर्क कर सकते हैं। रिपोर्ट सबमिट करने के तरीके के बारे में अधिक विस्तृत निर्देशों के लिए, जिसमें गुमनाम रूप से रिपोर्ट सबमिट करने का तरीका भी शामिल है, कृपया हमार [घटना समाधान प्रक्रियाएँ](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md) देखें। आप तीन व्यावसायिक दिनों के भीतर जवाब प्राप्त करने की उम्मीद कर सकते हैं।
 
-Linux Foundation द्वारा निर्मित CNCF इवेंट में होने वाली घटनाओं के लिए कृपया संपर्क करें eventconduct@cncf.io.
+Linux Foundation द्वारा निर्मित CNCF इवेंट में होने वाली घटनाओं के लिए, कृपया [eventconduct@cncf.io](mailto:conduct@cncf.io) पर संपर्क करें।
```

Comment:
translation needed here for लिनक्स फाउंडेशन (Linux Foundation), I guess we can also write like this to keep both सीएनसीएफ (CNCF)

Reply:
पूर्ण वीरान (Full Stop) is missing in most of the ending statements